Formula/mujs.rb

```diff
@@ -1,13 +1,16 @@
 class Mujs < Formula
   desc "Embeddable Javascript interpreter"
   homepage "https://www.mujs.com/"
-  # use tag not tarball so the version in the pkg-config file isn't blank
-  url "https://github.com/ccxvii/mujs.git",
-      tag:      "1.3.3",
-      revision: "57e3f01d5f29c5823be725d96284488edf5f8ae1"
+  url "https://mujs.com/downloads/mujs-1.3.3.tar.gz"
```

Comment:
Would you mind adding a unit test of the version inside the pkg-config file, previous issue https://github.com/Homebrew/homebrew-core/issues/21509/